generators/add-web-assets/exc-react/templates/src/App.js

```diff
@@ -41,6 +41,25 @@ export default class App extends React.Component {
 
     console.log('runtime object:', this.props.runtime)
     console.log('ims object:', this.props.ims)
+
+    // respond to configuration change events (e.g. user switches org)
+    this.props.runtime.on('configuration', ({ imsOrg, imsToken, locale }) => {
+      console.log('configuration change', { imsOrg, imsToken, locale })
+      // if (imsOrg !== this.state.imsOrg) {
```

Comment:
delete commented out code?

Reply:
the purpose of this handler now is just logging?

Reply:
same as above just showing an example, but yes we can argue that those comments might be confusing, are you in favor for deleting the commented code ?

Reply:
yeah, delete. we've got source code control nowadays to view history 😅

Reply:
what do you mean by source code control? That comment was to show to the user how to use the handler in the context of the generated template :)
But yeah will delete

Reply:
d'oh! forgot this was a generator.... 🤦

Reply:
that's like inception code 😄